src/com/insightdatascience/xlnet_learn2learn/data_pipeline/scripts/model_inference.sh

```diff
@@ -0,0 +1,27 @@
+#### local path
```

Comment:
specify this is a bash scriipt by including "#!/bin/bash" in the first line of the .sh file